spec/models/miq_ae_field_spec.rb

```diff
@@ -72,7 +72,23 @@
 
     it "doesn't access database (either via name uniqueness or set_user_info validation) when unchanged model is saved" do
       miq_ae_field = described_class.create!(:name => "display_name")
-      expect { miq_ae_field.save }.to make_database_queries(:count => 2)
+      expect { miq_ae_field.valid? }.not_to make_database_queries
+    end
+
+    describe "name validation" do
+      it "doesn't allow spaces" do
```

Comment:
the other test has me concerned now

can you pass in a nil or blank name and make sure it blows up?

thnx